apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx

```diff
@@ -33,27 +34,46 @@ const ComponentStatus = () => {
     components,
   } = data.allComponentStatusYaml.edges[0].node
 
+  const { Head, Body, Row, Cell } = Table
+
   return (
     <Table summary={summary}>
-      <thead>
-        <Tr>
-          {headers.map((text, index) => (
-            <ThH key={camelify(text)} scope="col">
+      <Head>
+        <Row>
+          {headers.map((text) => (
+            <Cell
+              as="th"
+              style={{ textAlign: 'left' }}
```

Comment:
We fixed this so should work without this prop :D

Reply:
Removed in #455